src/Publish/Microsoft.NET.Sdk.Publish.Tasks/WebConfigTransform.cs

```diff
@@ -150,17 +150,17 @@ private static void TransformAspNetCore(XElement aspNetCoreElement, string appNa
             // Set the hostingmodel attribute only if it is not already set in the web.config and AspNetCoreHostingModel property is set.
             if (hostingModelAttributeValue == null && !string.IsNullOrEmpty(aspNetCoreHostingModel))
             {
-                switch(aspNetCoreHostingModel.ToLower())
+                switch(aspNetCoreHostingModel.ToUpperInvariant())
```

Comment:
It is suggested to use String.Equals(a, b, StringComparison.InvarianCultureIgnoreCase) for such things, i believe..

Reply:
This is specifically a backport of the fix. I dont want to change the code across branches. We could make that change in master if we need.